modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-service/bnd.bnd

```diff
@@ -14,6 +14,6 @@ Import-Package:\
 	*
 Liferay-Releng-Module-Group-Description:
 Liferay-Releng-Module-Group-Title: Dynamic Data Lists
-Liferay-Require-SchemaVersion: 1.1.4
+Liferay-Require-SchemaVersion: 2.0.0
```

Comment:
@pedroqueiroz94 Why did you use 2.0.0? why not 1.1.5?

Reply:
@leoadb I used 2.0.0 because it's a huge change. Should we just use 1.1.5?